### PR TITLE
Fix list view

### PR DESCRIPTION
**Pull request by todd-dembrey**
Updated the tagged_list_view to work with versions of django>1.7 and added tests.

@incuna/backend 

``` terminal
  File "/home/todd/VirtualEnv/feincms-articles/lib/python3.5/site-packages/taggit/views.py", line 20, in tagged_object_list
    return ListView.as_view(request, qs, **kwargs)
TypeError: as_view() takes 1 positional argument but 3 were given
```

### CODE REVIEW

File: tests/templates/tests/food_list.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (Minglee01) [inline tests/templates/tests/food_list.html]: i think you need a newline here.

Comment (todd-dembrey) [inline tests/templates/tests/food_list.html]: Its a template for test purposes... should probably be blank

File: tests/templates/tests/food_tag_list.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```
Comment (Minglee01) [inline tests/templates/tests/food_tag_list.html]: think you need a space here

File: tests/urls.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from .tests import FoodTagListView
+
+urlpatterns = [
+    url(r'^food/tags/(?P<slug>[a-z0-9_-]+)/$', FoodTagListView.as_view(),
+        name='food-tag-list'),
+]
```
Comment (Minglee01) [inline tests/urls.py]: new line here.

File: tests/urls.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from .tests import FoodTagListView
+
+urlpatterns = [
+    url(r'^food/tags/(?P<slug>[a-z0-9_-]+)/$', FoodTagListView.as_view(),
+        name='food-tag-list'),
```
Comment (Minglee01) [inline tests/urls.py]: i think if we split it partially we split it all up

**Comment by Minglee01** [comment]
looks good but some layout issues i think.
ill defer to @incuna/backend

**Pull request closed without merging**